client/app/main/warmupModal.html

```diff
@@ -1,10 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>warmup</title>
+
 <!--<div>Pre-dream warm-up text here.</div>-->
-<img class="community-image" src="assets/images/community.png"/>
-<!--<div>Dream Brainstorm
-Take 5-8 minutes to put your dreams and goals on paper.  Feel free to brainstorm with lists, drawings, or figures etc., whatever you need to get the ideas flowing. This will be yours to keep and private to you. Your dreams may look very different today then tomorrow and that is okay. You’re just trying to get the ideas flowing and on paper.
+</head>
+     <div class="texty">
+ 	<h3 id="title2"> Affirm and Inspire </h3>
+    <body id="textBody">
```

Comment:
you probably meant to use a `<section>` tag here? Also, make sure to close all tags or the site will start to look buggy. `<section></section>`